lib/tasks/publishing_api.rake

```diff
@@ -8,4 +8,15 @@ namespace :publishing_api do
 
     PublishingApiFinderPublisher.new(finder_loader.finders).call
   end
+
+  # Find the manual by base_path from content store.
+  # Publish a redirect for the manual.
+  # Iterate through links[sections], publish a redirect per section.
+  desc "Redirect manual and sections"
```

Comment:
It might be useful to condense some of this comment into the task description, given the current task description is effectively just the name of the task, which isn't all that useful.